.github/workflows/check.yml

```diff
@@ -2,37 +2,27 @@ name: 'Check'
 run-name: 'Test, build, lint and validate'
 on:
   pull_request:
-jobs:
 
-  test:
-    runs-on: 'ubuntu-latest'
-    name: 'Checking code and dependencies'
+jobs:
+  build-test-lint-validate:
+    runs-on: ubuntu-latest
```

Comment:
Minor thing: It would be nice if the use of single-quotes for string values is consistent. Personally I am in favor of having them wrapped in single-quotes :)